packages/astro/src/assets/runtime.ts

```diff
@@ -71,7 +39,7 @@ type SvgAttributes = Record<string, any>;
  * Some attributes required for `image/svg+xml` are irrelevant when inlined in a `text/html` document. We can save a few bytes by dropping them.
  */
 const ATTRS_TO_DROP = ['xmlns', 'xmlns:xlink', 'version'];
-const DEFAULT_ATTRS: SvgAttributes = { role: 'img' };
+const DEFAULT_ATTRS: SvgAttributes = {};
```

Comment:
Just calling out this change! Probably the right move given the other changes, but I'd still like to provide good accessibility defaults where we can.

Reply:
For sure, we definitely need to follow up on this. Just removing for the initial release as there isn't consensus on what the best approach is yet